pkg/cmd/edge_applications/publish/get_file_list.go

```diff
@@ -0,0 +1,52 @@
+package publish
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/go-git/go-git/v5/plumbing/format/gitignore"
+)
+
+// Return a list of files inside the given directory, ignoring the given globs
+func (cmd *PublishCmd) getFileList(root string, ignore_globs []string) ([]string, error) {
+	var files []string
+
+	// parse ignore globs with gitignore.Pattern.ParsePattern
```

Comment:
do you believe an explanation to the user is necessary regarding this pattern? or we just assume they'll know?

Reply:
good point! I'll add something

Reply:
Perfect!! I'm doing some local tests with this branch now :)